bin/setup-test-projects.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this bash script if the same can be achieved by exportProjectsInTestResources?